src/qibo/backends/abstract.py

```diff
@@ -355,6 +356,21 @@ def calculate_matrix_exp(
         """
         raise_error(NotImplementedError)
 
+    @abc.abstractmethod
+    def calculate_matrix_power(
+        self, matrix, power: Union[float, int]
+    ):  # pragma: no cover
+        """Calculates the (fractional) ``power`` :math:`\\alpha` of ``matrix`` :math:`A`,
```

Comment:
```suggestion
        """Calculate the (fractional) ``power`` :math:`\\alpha` of ``matrix`` :math:`A`,
```

Reply:
I usually refer to [PEP 257](https://peps.python.org/pep-0257/#one-line-docstrings).